spec/exercise_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+describe Bibliotheca::Exercise do
+  let(:json) {{
+        type: 'problem',
+        name: 'Multiple',
+        editor: 'multiple_choice',
+        choices: [{value: 'foo', checked: true}, {value: 'bar', checked: false}],
```

Comment:
It would be nice to have an **additional** scenario where multiple values are checked. Otherwise, this test is no different of a single choice test.